storm-core/src/jvm/org/apache/storm/scheduler/SchedulerAssignment.java

```diff
@@ -55,4 +56,6 @@
     public Set<ExecutorDetails> getExecutors();
     
     public Set<WorkerSlot> getSlots();
+
+    public Map<WorkerSlot, Collection<ExecutorDetails>> getSlotToExecutors();
```

Comment:
Up to you. It's a really minor thing, and this might get resolved when we adopt a Java standard style later. I am fine if you want to leave it in or change it, either way.